3d-map/webpack.common.js

```diff
@@ -44,7 +47,18 @@ module.exports = {
       inject: false,
       // favicon: 'assets/img/favicon.ico'
     }),
+    new CopywebpackPlugin({ patterns: [{ from: path.resolve(path.join(cesiumSource, cesiumWorkers)), to: 'Workers' }] }),
```

Comment:
Oh, that's the trick! 😮

Reply:
Its just for the assets loaded during runtime, but actualy I was struggling to get it to work until I changed this line https://github.com/hslayers/examples/pull/4/files/5203e5f0a961f846f3e0cfc74bc602e51ebdf140..d8d5ee13285ed13b2e073218aede8d322088950d#diff-5cd1e29f182718e6bd65f2c37e4b3b79R36